src/world.rs

```diff
@@ -83,6 +92,55 @@ impl World {
         self.flush();
 
         let entity = self.entities.alloc();
+
+        self.spawn_inner(entity, components);
+
+        entity
+    }
+
+    /// Create an entity with certain components with a specific Entity handle.
+    ///
+    /// See `spawn`.
+    ///
+    /// This will overwrite an existing entity, if the entity IDs are the same.
+    ///
+    /// Spawning with an older Entity handle, will cause other copies of the same Entity
+    /// handles to become live again.
+    ///
+    /// Be cautious with using this functionality to resurrect old entity handles as it
+    /// will vastly increase the likelihood of encountering the aforementioned issue
+    /// of dead entity handles erroneously becoming live again.
+    ///
+    /// # Example
+    /// ```
+    /// # use hecs::*;
+    /// let mut world = World::new();
+    /// let a = world.spawn((123, "abc"));
+    /// let b = world.spawn((456, true));
+    /// world.despawn(a);
+    /// assert!(!world.contains(a));
+    /// // all previous references to 'a' will be valid again, but pointing to the new entity 'c'.
```

Comment:
```suggestion
    /// // all previous references to 'a' will be valid again, but pointing to the new entity.
```
There is no 'c' introduced.